errors.go

```diff
@@ -113,6 +113,12 @@ func Wrap(cause error, message string) error {
 	}
 }
 
+// Wrapf returns an error annotating the cause with the format specifier.
+// If cause is nil, Wrapf returns nil.
+func Wrapf(cause error, format string, args ...interface{}) error {
+	return Wrap(cause, fmt.Sprintf(format, args...))
```

Comment:
Please add a test to ensure that the correct line number is being recorded. I think this code will show that Wrapf is the owner, not it's caller